CHANGELOG.md

```diff
@@ -1,6 +1,29 @@
 # CHANGELOG
 
 -----
+
+## `v8.1.0-beta`
+### ARM
+| api                                 | version            | note                               |
+|:------------------------------------|:-------------------|:-----------------------------------|
+| arm/apimanagement                   | 2016-07-07         | new                                |
+| arm/apideployment                   | 2016-07-07         | new                                |
+| arm/billing                         | 2017-02-27-preview | new                                |
+| arm/compute                         | 2016-04-30-preview | update to latest swagger           |
+| arm/containerservice                | 2017-01-31         | update to latest swagger           |
+| arm/customer-insights               | 2017-01-01         | new                                |
+| arm/graphrbac                       | 1.6                | new                                |
```

Comment:
Currently we can gen comp swaggers, but the generated code is not usable.
I just gen one of the swaggers. It should work just as the other parts of the SDK.